staging/src/k8s.io/api/scheduling/v1alpha1/types.go

```diff
@@ -166,13 +166,6 @@ type PodGroup struct {
 
 // PodGroupPolicy defines the scheduling configuration for a PodGroup.
 type PodGroupPolicy struct {
-	// Basic specifies that the pods in this group should be scheduled using
-	// standard Kubernetes scheduling behavior.
-	//
-	// +optional
-	// +oneOf=PolicySelection
-	Basic *BasicSchedulingPolicy `json:"basic,omitempty" protobuf:"bytes,2,opt,name=basic"`
```

Comment:
What is the meaning of a Workload with a PodGroup with no policy?  Is it meant to be allowed?  As it stands, it would be an error -- do we want this Workload to be able to describe non-gang workloads?  It seems to me that we would want that, but I may be wrong.

Mechanically - we do not want "no value" to have meaning because that prevent future evolution of the API (hostile to version-skew).  So if we want to be allowed to represent non-gang workloads, we need SOMETHING here to be explicit.  

We can always add `basic` later, but it will have to go through its own alpha->beta->GA cycle.